commit 4f2dd4fe0c28102f005b0070e30396a3a5488968
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-18T11:22:21-04:00 (7 months ago)
    
    TOOLS-2207 eng submodule should use https (not git@)
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index 3927626..bc6f366 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -6,4 +6,4 @@
 	url = git://github.com/davepacheco/jsstyle.git
 [submodule "deps/eng"]
 	path = deps/eng
-	url = git@github.com:joyent/eng.git
+	url = https://github.com/joyent/eng.git
